GoogleSignIn/Tests/Unit/GIDGoogleUserTest.m

```diff
@@ -155,12 +154,12 @@ - (void)testUpdateAuthStateWithUnchangedRefreshTokenAndIDToken {
 
 #pragma mark - Helpers
 
-- (GIDGoogleUser *)googleUserWithAccessTokenExpireTime:(NSTimeInterval)accessTokenExpireTime
-                                     idTokenExpireTime:(NSTimeInterval)idTokenExpireTime {
+- (GIDGoogleUser *)googleUserWithAccessTokenExpiresIn:(NSTimeInterval)accessTokenExpiresIn
+                                    idTokenExpireTime:(NSTimeInterval)idTokenExpireTime {
```

Comment:
Should we make ID token expiration consistent with access token here?

Reply:
I prefer to make it simple. The less back and forth conversion, the less likely there are mistakes.

Reply:
I don't feel strongly about it, as it's just a test helper, but I think it might be easier to work with if the signature were:

```
- (GIDGoogleUser *)googleUserWithAccessTokenExpiresIn:(NSTimeInterval)accessTokenExpiresIn
                                     idTokenExpiresIn:(NSTimeInterval)idTokenExpiresIn
```

Reply:
That's a brilliant idea.